src/main/java/net/pms/formats/v2/AudioAttribute.java

```diff
@@ -1,5 +1,5 @@
 /*
- * PS3 Media Server, for streaming any media to your PS3.
+ * PS3 Media Server, for streaming any medias to your PS3.
```

Comment:
@Sami32 It's just my interpretation, but unless I'm mistaken the license rules says that you're not allowed to edit the header of a file that you modify (except to add information about the additional work done below the original message). It's the "rule" I follow at least, and it does make some sense. If someone for example want to search for every file containing the PMS header, it's nice that the header is the same everywhere. This is the way the header is in the "parent" PMS project remember.

I absolutely agree with you about correcting a mistake when you find it, it just think that license headers are a "special case" that should not be edited by anyone else than the original author(s).